app/controllers/admin/users_controller.rb

```diff
@@ -13,6 +13,25 @@ def index
       end
     end
 
+    def new
+      @user = User.new
+    end
+
+    def create
+      user_params = params.require(:user)
+      user_type = user_params[:type].camelize.downcase
+      case user_type
+      when 'enduser'
+        user_params[:type] = 'EndUser'
+      when 'adminuser'
+        user_params[:type] = 'AdminUser'
+      else
+        user_params[:type] = nil
+      end
+      @user = User.create(user_params.permit(*DEFAULT_FIELDS))
```

Comment:
This doesn't need to be an instance variable, you can use `user` instead of `@user`

Reply:
Removing the `@user` seems to create errors with regards to flash interpolation